CONTRIBUTING.md

```diff
@@ -26,26 +26,156 @@
+|Workflow status          |status-wontfix          |Any issue that it is determined will not be fixed.                                                                                                            |                                                                                              |
+|Tracking upstream project|tracking-upstream       |For tracking issues with upstream projects that are affecting the RPCO project.                                                                               |                                                                                              |
+|Product feature          |feature-gating          |Gating-related work.                                                                                                                                          |                                                                                              |
+|Product feature          |feature-upgrades        |Upgrades-related work.                                                                                                                                        |                                                                                              |
+|Product feature          |feature-monitoring      |Monitoring-related work.                                                                                                                                      |                                                                                              |
+|Product feature          |feature-osa             |OSA-related work.                                                                                                                                             |                                                                                              |
+
+## Pre-Backlog -> Backlog
+### Adding an issue to the backlog
+Click [here](https://github.com/rcbops/rpc-openstack/issues/new) to load the form used when creating a new issue in the backlog. The form is generated from [this template](.github/ISSUE_TEMPLATE.md) which includes a set of questions for the reporter and a checklist to be completed by the owner as the issue is worked. Currently Waffle does not support the use of an issue template and so issues should not be created directly in Waffle.
+
+## Backlog
+All issues that are are not approved to be worked are listed in Backlog, this includes new issues and those that have been triaged but not yet committed to.
+
+### Issue triage
+Issue triage is an ongoing process that should happen on a daily basis, the engineering team will triage new issues and monitor unprioritised issues so that new commitments are always targetting the most import work.
```

Comment:
Currently there is one weekly meeting. I would like to see more emphasis on triaging issues as they come in.